documentation/security/ThreatModel.rst

```diff
@@ -124,60 +128,60 @@ The previous schema gives only some examples of GlobaLeaks’s flexibility; but
 
 Data Security Matrix
 ====================
-This section highlights the data that is handled by GlobaLeaks software and how different protection schemes are applied to GlobaLeaks handled data.
+This section highlights the data that is handled by GlobaLeaks and how different protection schemes are applied to GlobaLeaks handled data.
 
 The following data are the one involved within GlobaLeaks:
 
 .. csv-table::
    :header: "Data", "Description"
 
-   "Submission data", "Those are the data associated with a submission such as the filled forms and selectors provided by the Whistleblower."
-   "Submission files", "Those are the files associated with a submission that may require to be handled with special care due to per recipient’s encryption and optional metadata cleanup."
```

Comment:
`Submission files` definition needs an edit, but I'm not confident to do so. The exact meaning of `per recipient’s encryption and optional metadata cleanup` is unclear to me.

Reply:
Actually here i would just simplify to: "Those are the files associated with a submission."